appservice/src/deleteSite/IDeleteSiteWizardContext.ts

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { ExecuteActivityContext, IActionContext } from "@microsoft/vscode-azext-utils";
+import { ParsedSite } from "../SiteClient";
+
+export interface IDeleteSiteWizardContext extends IActionContext, ExecuteActivityContext {
```

Comment:
In [Nathan's PR](https://github.com/microsoft/vscode-azurefunctions/pull/3118/files#diff-ebdac05616fdc2bd7e3e037cbf5b4e6899f00a1a0742acd7045f066825b4612e) there's a similar interface that also includes `resourceGroupToDelete`, it may be helpful to combine these two in this PR where they can be shared.